dask/bytes/core.py

```diff
@@ -19,8 +19,9 @@
 _open_text_files = dict()
 
 
-def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
-                     sample=True, compression=None, **kwargs):
+def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
+                     sample=True, compression=None, storage_options=None,
+                     **kwargs):
```

Comment:
`storage_options` is used in this function extending it with information extracted from `urlpath`:

```python
storage_options = infer_storage_options(urlpath, storage_options)
```